src/print.c

```diff
@@ -53,6 +54,20 @@ int apk_get_screen_width(void)
 	return apk_screen_width;
 }
 
+char apk_get_human_unit(size_t size, double *dest)
```

Comment:
> And use it also in apk_solver_commit_changeset() around src/commit.c:286

Done.

> Please make the input size signed.

Why?

Reply:
Because `apk_solver_commit_changeset()` can feed it negative values when e.g. removing packages, or upgrading and the new package is smaller.

Reply:
So the `for` loop comparison for `s >= 1024` should likely be `fabs(s) >= 1024` or similar to allow it  functioning right for negative values.

Reply:
Oh. Seems you fixed caller instead. I guess that is good enough too.

Reply:
> Because apk_solver_commit_changeset() can feed it negative values […]

I worked around that by invoking the function as `apk_get_human_unit(labs(size_diff), &humanized);` see `29f7036e56198ccf1f61f4174d5000e041cc2b27`. Shouldn't that work as well or am I missing something here?